packages/cozy-client/examples/web/src/Example.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react'
+import { useQuery, Q } from 'cozy-client'
+
+const styles = {
+  data: {
+    maxHeight: 200,
+    overflow: 'scroll',
+    background: 'lightyellow',
+    padding: '1rem'
+  },
+
+  stateTester: {
+    background: 'lavender',
+    padding: '1rem'
+  },
+
+  queries: {
+    display: 'flex'
+  },
+
+  block: {
+    border: '3px dotted',
+    margin: '1rem',
+    width: '50%',
+    padding: '1rem'
+  },
+
+  amount: {
+    positive: {
+      color: 'teal'
+    },
+    negative: {
+      color: 'crimson'
+    }
+  }
+}
+
+const accountsConn = {
+  as: 'accounts',
+  query: () => Q('io.cozy.bank.accounts')
+}
+
+const transactionsConn = {
+  as: 'transactions',
+  query: () => Q('io.cozy.bank.operations').limitBy(10).sortBy([{'date': 'desc'}]).where({})
+}
+
+const QueryResult = ({ collection }) => {
+
+  return (
+    <div style={styles.data}>
+      id: { collection.id }<br/>
+      bookmark: {collection.bookmark }<br/>
+      length: {collection.data ? collection.data.length : 0}{' '}
+    </div>
+  )
+}
+
+const useCounter = initialCount => {
+  const [count, setCount] = useState(initialCount)
+  const increment = () => setCount(count + 1)
+  const decrement = () => setCount(count - 1)
```

Comment:
```suggestion
  const increment = useCallback( () => setCount(count => count + 1), [setCount] )
  const decrement = useCallback( () => setCount(count => count - 1), [setCount] )
```

Not important here because this is an example, but it may help having the good habits. This change allows to return constant callbacks